### PR TITLE
Updates for 1.0.6/1.1.3 release

### PR DESCRIPTION
**Pull request by MichaelSimons**
This is a migration of the changes that already exist in microsoft/dotnet-nightly.

### CODE REVIEW

File: build-pipeline/dotnet-docker-linux-amd64-images.json [hunk 1]
```diff
@@ -122,7 +122,7 @@
       "allowOverride": true
     },
     "image-builder.imageName": {
-      "value": "microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20170807153845"
+      "value": "microsoft/dotnet-buildtools-prereqs:image-builder-jessie-20170817154316"
```
Comment (ravimeda) [inline build-pipeline/dotnet-docker-linux-amd64-images.json]: `20170817154316` - what produces this timestamp? Or how does one verify if this is correct?

**Comment by MichaelSimons** [comment]
@dotnet-bot test NanoServer - 1.1 Dockerfiles please

**Comment by dleeapho** [comment]
LGTM

File: build-pipeline/dotnet-docker-linux-amd64-images.json (hunk 1 above)
Comment (MichaelSimons) [reply build-pipeline/dotnet-docker-linux-amd64-images.json]: The timestamp is for when the image was built.  You can find the available tags at https://hub.docker.com/r/microsoft/dotnet-buildtools-prereqs/tags/.  The best way to verify this change is to compare against nightly which is our staging environment.

Comment (ravimeda) [reply build-pipeline/dotnet-docker-linux-amd64-images.json]: Thanks Michael.

**Comment by MichaelSimons** [comment]
Closing as this PR is no longer applicable.

**Pull request closed without merging**